.github/workflows/ci.yml

```diff
@@ -0,0 +1,65 @@
+---
+
+name: CI
+
+on:
+  push:
+  pull_request:
+
+env:
+  galaxy-name: "cvmfs_contrib.cvmfs_client"
+
+jobs:
+
+  molecule:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        distro: [centos8]
+      fail-fast: false
+
+    steps:
+
+    - uses: actions/checkout@v2
+      with:
+        path: ${{ env.galaxy-name }}
+
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+
+    - name: Upgrade pip
+      run: |
+        pip install --upgrade pip wheel
+        pip --version
+
+    - name: Install requirements
+      run: |
+        pip install --use-feature=2020-resolver -r requirements.txt
+      working-directory: ${{ env.galaxy-name }}
+
+    # See https://github.com/geerlingguy/raspberry-pi-dramble/issues/166
+    - name: Force GitHub Actions' docker daemon to use vfs.
+      run: |
+        sudo systemctl stop docker
+        echo '{"cgroup-parent":"/actions_job","storage-driver":"vfs"}' | sudo tee /etc/docker/daemon.json
+        sudo systemctl start docker
+
+    - name: Run molecule
+      run: molecule test
+      working-directory: ${{ env.galaxy-name }}
+      env:
+        MOLECULE_DISTRO: ${{ matrix.distro }}
+
+  # release:
```

Comment:
This would make it straightforward to publish this role to https://galaxy.ansible.com/ (released automatically when pushing a tag).

I can remove this again if it is not desired